stdlib/test/collections/test_list.mojo

```diff
@@ -589,6 +589,12 @@ def test_converting_list_to_string():
     assert_equal(__type_of(my_list3).__str__(my_list3), "[1.0, 2.0, 3.0]")
 
 
+def test_list_count():
```

Comment:
**Suggestion** I recommend adding a few more test cases:
- `count()` on an empty list
- `count()` when there are no matches to ensure it returns `0`

Reply:
Updated

Reply:
this test case is covered below on line 596 @gabrieldemarmiesse

Reply:
oupsy, my bad. Ignore my comment then